File: eBotanika.Model/Ocjena.cs
namespace eBotanika.Model
{
    public class Ocjena
    {
        public int OcjenaID { get; set; }
        public decimal OcjenaUsluge { get; set; }
        public int KorisnikID { get; set; }
        public Korisnik Korisnik { get; set; }
        public int BiljkeID { get; set; }
        public Biljke Biljke { get; set; }
    }
}

File: ebotanika_mobile/lib/models/ocjena.dart
class Ocjena {
  int ocjenaID;
  double ocjenaUsluge;
  int korisnikID;
  int biljkeID;

  Ocjena(
      {required this.ocjenaID,
      required this.ocjenaUsluge,
      required this.korisnikID,
      required this.biljkeID});

  factory Ocjena.fromJson(Map<String, dynamic> json) {
    return Ocjena(
        ocjenaID: json["ocjenaID"],
        ocjenaUsluge: json["ocjenaUsluge"],
        korisnikID: json["korisnikID"],
        biljkeID: json["biljkeID"]);
  }

  Map<String, dynamic> toJson() => {
        "ocjenaID": ocjenaID,
        "ocjenaUsluge": ocjenaUsluge,
        "korisnikID": korisnikID,
        "biljkeID": biljkeID
      };
}
